src/app/infrastructure/shared/config/app.config.ts

```diff
@@ -0,0 +1,25 @@
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+export const getConfig = (dev: any, prod: any, stage: any) => {
+    let envVars: any;
+
+    if (Object.prototype.hasOwnProperty.call(process.env, 'environment') &&
+        process.env.environment
+    ) {
+        switch (process.env.environment) {
+            case 'prod':
+                envVars = prod;
```

Comment:
Without additional processing on the `envVars` string, why not just short-circuit the function and return instead of assignment?

Reply:
Good. I did apply your suggestions.